front-end/src/index.js

```diff
@@ -5,10 +5,7 @@ import App from './components/App';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <App />
-    
-  </React.StrictMode>
+  <App />
```

Comment:
Put this back you bastard

Reply:
This is not what I meant when I said "Fix the linting errors"

Reply:
added back in